README.md

```diff
@@ -0,0 +1,2 @@
+# CrabigatorBot
+Discord bot made for the WaniKani AI Mnemonic project.
```

Comment:
I'm leaning more toward Mnemonic Images rather than AI Mnemonic since some images are hand-drawn or created by other means. Also, we should be careful with how we use the WaniKani trademark, we don't want to seem like any kind of official project.

Reply:
Okay, I'm also thinking something along the lines of "Image Mnemonics Project". And, I also called it an unofficial WaniKani (...) project.